templates/registration/register.html.twig

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="{{ app.request.get('_locale') }}">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>{{ 'register.title'|trans }} • {{ 'app.name'|trans }}</title>
+
+    <!-- Global stylesheets -->
+    <link href="https://fonts.googleapis.com/css?family=Roboto:400,300,100,500,700,900" rel="stylesheet"
+          type="text/css">
+    {{ encore_entry_link_tags('app') }}
+    <!-- /global stylesheets -->
+</head>
+
+<body class="login-container login-cover"
+      style="background: url({{ asset('images/backgrounds/login.jpg') }}) no-repeat; background-size: cover;">
+<!-- Page container -->
+<div class="page-container">
+    <!-- Page content -->
+    <div class="page-content">
+        <!-- Main content -->
+        <div class="content-wrapper">
+            <!-- Content area -->
+            <div class="content pb-20">
+                <!-- Form with validation -->
+                <div class="panel panel-body login-form">
+                    <div class="text-center mb-3">
+                        <div class="icon-object border-warning-400 text-warning-400"><i class="icon-key"></i></div>
+                        <h5 class="content-group-lg">{{ 'register.title'|trans }}
+                            <small class="display-block">{{ 'register.subtitle'|trans }}</small>
+                        </h5>
+                    </div>
+                     {% for flash_error in app.flashes('verify_email_error') %}
+                        <div class="alert alert-danger" role="alert">{{ flash_error }}</div>
+                    {% endfor %}
+                    {{ form_errors(registrationForm) }}
```

Comment:
could you format errors in red as well?

![image](https://github.com/captain-coaster/captain-coaster/assets/1555765/106c8ded-3317-4631-9f03-96c237718f46)